src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -30,42 +31,44 @@ export const KarmaScore = ({score}: KarmaScoreProps) => {
         </View>
         <Text
           style={{
-            fontWeight: '500',
+            fontFamily: fonts.inter[500],
             fontSize: 24,
             lineHeight: 36,
-            marginRight: 8
+            marginRight: 8,
+            color: COLORS.white2
           }}>{`${score} Karma`}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style properties of the Text component have been updated. The `fontFamily` property has been added and the `color` property has been changed. These changes seem to be part of an effort to standardize the look and feel of the app. Please ensure that these changes do not conflict with other parts of the application and that they adhere to the design guidelines.

```diff
-            fontWeight: '500',
-            color: '#FFFFFF'
+            fontFamily: fonts.inter[500],
+            color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->